modules/apps/27-interchain-accounts/controller/keeper/genesis.go

```diff
@@ -12,9 +12,9 @@ import (
 // InitGenesis initializes the interchain accounts controller application state from a provided genesis state
 func InitGenesis(ctx sdk.Context, keeper Keeper, state genesistypes.ControllerGenesisState) {
 	for _, portID := range state.Ports {
-		if !keeper.HasCapability(ctx, portID) {
-			capability := keeper.BindPort(ctx, portID)
-			if err := keeper.ClaimCapability(ctx, capability, host.PortPath(portID)); err != nil {
+		if !keeper.hasCapability(ctx, portID) {
+			cap := keeper.BindPort(ctx, portID)
```

Comment:
Could you please rename this to `capability` to make the linter happy? :)

Reply:
Ty